docs/pages/guides/configuring-js-engines.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Configuring JS Engines
+sidebar_title: Configuring JS Engines
+description: A guide on configuring JS Engines for both Android and iOS in an Expo project.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+JavaScript engines are responsible for executing your application code and providing various features like memory management, optimization, and error handling. Expo projects, by default, use [Hermes](https://hermesengine.dev/) as the JavaScript engine (in SDK 47 and lower, the default was JSC). It is also possible to switch to other engines like JSC or V8 for both Android and iOS platforms, but not for web because the JavaScript engine is included in the web browser.
+
+### Configuring the `jsEngine` through Expo config
```

Comment:
```suggestion
## Configuring the `jsEngine` through Expo config
```

[In general, as per our writing style guide, top level headings on a page should start from H2](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings).